service/src/test/java/uk/gov/hmcts/reform/fpl/utils/OrderIssuedNotificationTestHelper.java

```diff
@@ -53,6 +54,17 @@ private OrderIssuedNotificationTestHelper() {
             "documentLink", jsonFileObject);
     }
 
+    public static Map<String, Object> getExpectedAllocatedJudgeParameters() {
```

Comment:
again here could return the java object no?

Reply:
changed :)